### PR TITLE
Add reminder to check tests in contributing.md

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,8 +167,10 @@ To send us a pull request:
 
 Please make sure all your checkins have detailed commit messages explaining the patch.
 
-Once you've gotten an LGTM from a project maintainer, submit your changes to the
-`master` branch using one of the following methods:
+Once you've gotten an LGTM from a project maintainer, verify that all of the automated
+tests on your pull request have successfully completed. You can see the results
```
Comment (goderbauer) [inline CONTRIBUTING.md]: tests -> checks? Github seems to call these things checks on the PR page.

**Review by goderbauer — APPROVED** [review]
LGTM, thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (xster) [inline CONTRIBUTING.md]: SG

**Comment by goderbauer** [comment]
I think we should go with https://github.com/flutter/flutter/pull/11190 instead, which removes the option to directly commit to master.

**Comment by xster** [comment]
oh ha, race condition. Closing.

**Comment by Hixie** [comment]
haha nice. i love you guys :-)

**Pull request closed without merging**